templates/rhpam73-managed.yaml

```diff
@@ -0,0 +1,1178 @@
+## KIE server services 1 END
+## PostgreSQL service 1 BEGIN
+- apiVersion: v1
+  kind: Service
+  metadata:
+    annotations:
+      description: The first database server's port.
+    labels:
+      application: ${APPLICATION_NAME}
+      service: "${APPLICATION_NAME}-postgresql-1"
+    name: ${APPLICATION_NAME}-postgresql-1
+  spec:
+    ports:
+    - port: 5432
+      targetPort: 5432
+    selector:
+      deploymentConfig: ${APPLICATION_NAME}-postgresql-1
+## PostgreSQL service 1 END
+- kind: Route
+  apiVersion: v1
+  id: "${APPLICATION_NAME}-rhpamcentrmon-http"
+  metadata:
+    name: "${APPLICATION_NAME}-rhpamcentrmon"
+    labels:
+      application: "${APPLICATION_NAME}"
+      service: "${APPLICATION_NAME}-rhpamcentrmon"
+    annotations:
+      description: Route for Business Central Monitoring's http service.
+      haproxy.router.openshift.io/timeout: 60s
+  spec:
+    host: "${BUSINESS_CENTRAL_HOSTNAME_HTTP}"
+    to:
+      name: "${APPLICATION_NAME}-rhpamcentrmon"
+    port:
+      targetPort: http
+- kind: Route
+  apiVersion: v1
+  id: "${APPLICATION_NAME}-rhpamcentrmon-https"
+  metadata:
+    name: "secure-${APPLICATION_NAME}-rhpamcentrmon"
+    labels:
+      application: "${APPLICATION_NAME}"
+      service: "${APPLICATION_NAME}-rhpamcentrmon"
+    annotations:
+      description: Route for Business Central Monitoring's https service.
+      haproxy.router.openshift.io/timeout: 60s
+  spec:
+    host: "${BUSINESS_CENTRAL_HOSTNAME_HTTPS}"
+    to:
+      name: "${APPLICATION_NAME}-rhpamcentrmon"
+    port:
+      targetPort: https
+    tls:
+      termination: passthrough
+## KIE server routes 1 BEGIN
+- kind: Route
+  apiVersion: v1
+  id: "${APPLICATION_NAME}-kieserver-1-http"
+  metadata:
+    name: "${APPLICATION_NAME}-kieserver-1"
+    labels:
+      application: "${APPLICATION_NAME}"
+      service: "${APPLICATION_NAME}-kieserver-1"
+    annotations:
+      description: Route for First KIE server's http service.
+  spec:
+    host: "${KIE_SERVER1_HOSTNAME_HTTP}"
+    to:
+      name: "${APPLICATION_NAME}-kieserver-1"
+    port:
+      targetPort: http
+- kind: Route
+  apiVersion: v1
+  id: "${APPLICATION_NAME}-kieserver-1-https"
+  metadata:
+    name: "secure-${APPLICATION_NAME}-kieserver-1"
+    labels:
+      application: "${APPLICATION_NAME}"
+      service: "${APPLICATION_NAME}-kieserver-1"
+    annotations:
+      description: Route for First KIE server's https service.
+  spec:
+    host: "${KIE_SERVER1_HOSTNAME_HTTPS}"
+    to:
+      name: "${APPLICATION_NAME}-kieserver-1"
+    port:
+      targetPort: https
+    tls:
+      termination: passthrough
+## KIE server routes 1 END
+- kind: DeploymentConfig
+  apiVersion: v1
+  metadata:
+    name: "${APPLICATION_NAME}-rhpamcentrmon"
+    labels:
+      application: "${APPLICATION_NAME}"
+      service: "${APPLICATION_NAME}-rhpamcentrmon"
+    annotations:
+      template.alpha.openshift.io/wait-for-ready: "true"
+  spec:
+    strategy:
+      type: Rolling
```

Comment:
Is there a specific reason to use Rolling instead Recreate?

Reply:
For KIE server pod, using Rolling is to have a non-interrupted experience during Kjar development. 
@spolti Do you see any potential issue with Rolling strategy?

Reply:
Updated rollingParams